### DISCUSSION TITLE
Standard example has 2 commas

### DISCUSSION THREAD

**Discussion by rowild** [Comfy-Org/ComfyUI#2421] [Q&A]
I am new to comfyUI. Having it set up on a Mac M2, I immediately see that there is already a prompt given. But that prompt has 2 commas:

`beautiful scenery nature glass bottle landscape, , purple galaxy bottle,`

Do the extra comma and extra space have a special meaning?

**Comment by ltdrdata** [comment]
It plays a role in clearly separating both sides when encoding by spreading the distance between tokens.

**Reply by rowild** [reply]
Thank you very much for your quick reply! However, I do understand the words of your reply, but not the meaning. What are the "both sides" in this case?
(I am new to this stuff; with your info I searched Google for "distance between tokens" and got some insight into that; but here the "two side" do not yet make sense to me – any help would be most welcome!)

**Reply by ltdrdata** [reply]
"beautiful scenery nature glass bottle landscape" and "purple galaxy bottle".
It mitigates the impact of each part's prompt on another prompt. Of course, it's not a perfect solution; it's just one technique.

Adding a comma implies inserting additional tokens, and having more tokens positioned between the front and back parts weakens the correlation.
